### PR TITLE
Add magichome document 

### PR DESCRIPTION
**Pull request by ZNEGGE-SDK**
**Description:**

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#28581

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/magichome.markdown [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+```yaml
+magichome:
+  username: YOUR_MAGICHOME_USERNAME
+  password: YOUR_MAGICHOME_PASSWORD
+  country_code: YOUR_ACCOUNT_COUNTRYCODE
+```
+
+{% configuration %}
+username:
```
Comment (klaasnicolaas) [inline source/_integrations/magichome.markdown:30]: Missing `platform` and `company`

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/magichome.markdown (hunk 1 above)
Comment (ZNEGGE-SDK) [reply source/_integrations/magichome.markdown:30]: OK ! My colleague has already made up.

**Comment by LIULi-VVET** [comment]
Hello @klaasnicolaas ! Is there anything else that needs to be modified?

**Review by klaasnicolaas — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged 👍
⚠️ The ha_release might need an update before merging this one in.

**Comment by klaasnicolaas** [comment]
@LIULi-VVET Since the PR is a bit old and we have added more checks in the meantime, it would be nice if you updated your branch on your fork and pushed to this PR. Otherwise we cannot merge when the PR is ready in the code repo.

**Comment by LIULi-VVET** [comment]
> @LIULi-VVET Since the PR is a bit old and we have added more checks in the meantime, it would be nice if you updated your branch on your fork and pushed to this PR. Otherwise we cannot merge when the PR is ready in the code repo.
OK!  Thank you very much !

**Pull request closed without merging**